qutebrowser/browser/downloads.py

```diff
@@ -286,6 +285,13 @@ def _ask_overwrite_question(self):
                                     window=self._win_id)
         message_bridge.ask(q, blocking=False)
 
+    def _download_dir(self):
+        """Get the download directory to use."""
+        download_dir = config.get('storage', 'download-directory')
+        if download_dir is None:
+            download_dir = standarddir.get(QStandardPaths.DownloadLocation)
```

Comment:
Small hint: If you want to rebase this on master (you don't have to!) that'd be `standarddir.download` now.